changelog.d/15820.bugfix

```diff
@@ -0,0 +1 @@
+Fix long-standing bug where remote invites weren't correctly pushed.
```

Comment:
Was this fix spurred on by anything? Context?

Reply:
Just noticed the error in Sentry when I was looking at other stuff